ansible/roles/iptables-persistent/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
   tags: install_iptables_persistent
```

Comment:
The `notify` in the install step seems needless. Can we also makes the `name` attribute in this playbook more readable?